### ISSUE TITLE
The color contrast in warning detail view should be improved

### ISSUE THREAD

**Issue by nucleus-ffm** [nucleus-ffm/foss_warn#117]
**Is your feature request related to a problem? Please describe.**
In the detailed view of a warning, some tags are difficult to read due to a poor contrast value

**Describe the solution you'd like**
Choose other colors with better contrast
